### ISSUE TITLE
Error Parsing Response: Malformed Response / Request Throttling

### ISSUE THREAD

**Issue by kanestapler** [noahc3/PokemonGoTeamManager#2]
Here is my screenshot. Paused at the end of log in script

![capture](https://cloud.githubusercontent.com/assets/1622388/17230590/a0962048-54e2-11e6-8298-61485c162efd.PNG)

Looked into it. The API throws an error at line #185 in the session.py

Block here: 
```
# Parse the default responses
    def parseDefault(self, res):
        try:
            logging.info('HERE', res)
            self._state.eggs.ParseFromString(res.returns[1])
            self._state.inventory.ParseFromString(res.returns[2])
            self._state.badges.ParseFromString(res.returns[3])
            self._state.settings.ParseFromString(res.returns[4])
        except Exception as e:
            logging.error(e)
            raise GeneralPogoException("Error parsing response. Malformed response")

        # Finally make inventory usable
        items = self._state.inventory.inventory_delta.inventory_items
        self.inventory = Inventory(items)
```

`res.returns` either doesn't exist or the array doesn't have that many items in it.

I'm not even sure if this is your apps problem, but it hasn't happened with any other work I've done with this API

**Comment by noahc3** [comment]
I've been able to replicate this while Pokemon GO's servers were unstable (they were going up and down constantly all day today). Can you try again now and tell me if it still occurs?

**Comment by noahc3** [comment]
From what I can tell, Niantic is starting to throttle requests made to their servers. I'll see if there is anything I can do to bypass it, otherwise may need to wait for the lovely's from /r/pokemongodev to come up with a solution.

**Comment by kanestapler** [comment]
Good find. I guess this is you :stuck_out_tongue: https://www.reddit.com/r/pokemongodev/comments/4v3twy/request_throttling/

**Comment by noahc3** [comment]
No thats someone else.

**Comment by noahc3** [comment]
Gonna try throwing a .5 second sleep in between each api call and see if that solves anything.

**Comment by noahc3** [comment]
@kanestapler Would you mind testing out a fix for me?

**Comment by bowb** [comment]
I would also be willing to test it out. Has it been checked into git?

**Comment by noahc3** [comment]
@bowb Yep, the only file that should need changing from the current version is /libs/py/pogoapi/pogotesting.py

**Comment by bowb** [comment]
@noahc3 I was able to log in and see data with the fix.

**Comment by noahc3** [comment]
@bowb Alright, thanks for the help!

**Comment by noahc3** [comment]
Fixed in [1.3.2 Beta 1](https://github.com/noahc3/PokemonGoTeamManager/releases/tag/1.3.2b1)
